src/components/sidebar/sidebar.css

```diff
@@ -0,0 +1,52 @@
+#header .pro-sidebar {
+    height: 90vh;
+}
+#header .pro-sidebar.collapsed {
+    width: 80px;
+    min-width: 80px;
+}
+
+#header .pro-icon {
+    font-size: 1.4rem;
+    font-weight: normal;
+}
+#header .pro-sidebar-inner {
+    background-color: transparent;
+}
+#header .pro-sidebar-inner .pro-sidebar-layout {
+    overflow-y: hidden;
+}
+#header .pro-sidebar-inner .pro-sidebar-layout .logotext p {
+    font-size: 20px;
+    padding: 0 20px;
+    color: #000;
```

Comment:
declare color in root directory use from there

Reply:
Yess.... I have made the changes